Common-Issues.md

```diff
@@ -89,9 +93,9 @@ Next, add EssentialsX as a `dependency` under `dependencies`:
 <dependencies>
     ...
     <dependency>
-        <groupId>net.ess3</groupId>
+        <groupId>net.essentialsx</groupId>
         <artifactId>EssentialsX</artifactId>
-        <version>2.17.2</version>
+        <version>2.19.0</version>
```

Comment:
`2.19.0` isn't out, this should be `net.essentialsx` and `2.19.0-SNAPSHOT` for dev builds or `net.ess3` and `2.18.2` for the previous release - see the EssentialsX README.